maintainer/CI/fix_style.sh

```diff
@@ -26,8 +26,13 @@ if ! git diff-index --quiet HEAD -- && [ "${1}" != "-f" ]; then
 fi
 
 if ! hash pre-commit 2>/dev/null; then
-    echo "pre-commit command not found."
-    exit 2
+    python3 -m pre_commit 2>&1 >/dev/null
```

Comment:
can we rely on interpreter name being ```python3```?

Reply:
Ideally, we would pull that from CMake, but the maintainer scripts are independent of the build system. However, there is a PEP that guarantees names like `python` for Python 2 and `python3` for Python 3 and that `python` is never Python 3. Most notably, this guarantees you get the same Python interpreter that you get when you run `pip3`.